README.md

```diff
@@ -31,15 +31,15 @@ kubectl apply -f https://github.com/fybrik/data-movement-operator/releases/lates
 
 ### Version compatbility matrix
 
-| Fybrik           | CBM     | Command
-| ---              | ---     | ---
-| 0.5.x            | 0.5.x   | `https://github.com/fybrik/data-movement-operator/releases/download/v0.5.0/modules/implicit-copy-batch-module.yaml`
-| master           | master  | `https://raw.githubusercontent.com/fybrik/data-movement-operator/master/modules/implicit-copy-batch-module.yaml`
-
-| Fybrik           | CSM     | Command
-| ---              | ---     | ---
-| 0.5.x            | 0.5.x   | `https://github.com/fybrik/data-movement-operator/releases/download/v0.5.0/modules/implicit-copy-stream-module.yaml`
-| master           | master  | `https://raw.githubusercontent.com/fybrik/data-movement-operator/master/modules/implicit-copy-stream-module.yaml`
+| Fybrik           | CBM     | Mover   | Command
```

Comment:
Please add an explanation what is CSM and CBM (copy-stream-module and copy-batch-module)